src/psyclone/domain/lfric/lfric_invoke.py

```diff
@@ -93,16 +93,16 @@ def __init__(self, alg_invocation, idx, invokes):
 
         # Import here to avoid circular dependency
         # pylint: disable=import-outside-toplevel
-        from psyclone.dynamo0p3 import (DynStencils,
-                                        DynFunctionSpaces, DynDofmaps,
+        from psyclone.dynamo0p3 import (DynStencils, DynFunctionSpaces,
                                         DynLMAOperators, DynGlobalSum,
                                         DynCMAOperators, DynBasisFunctions,
                                         DynMeshes, DynBoundaryConditions,
                                         DynProxies, DynCellIterators,
                                         DynReferenceElement,
                                         LFRicMeshProperties)
         from psyclone.domain.lfric import (LFRicLoopBounds, LFRicRunTimeChecks,
```

Comment:
Can you check if with your nice refactoring these imports (esp. from domain.lfric) can be moved to the top now? I quickly tried (not with the whole test suite), and moving the lfric ones up seems to work now (and if it does: thanks a lot for your refactoring work, really appreciated!!)

Reply:
OK, I just realised I used the wrong directory with the wrong branch ;) But still, I could move the older domain.lfric imports, so chances are that it might work here as well.

Reply:
Haha! Unfortunately, none the domain.lfric imports seem to work at the top level. I did try to run the whole test suite with each but they all failed before starting.

Reply:
I am getting the feeling that we (as the team) should revisit this - we might get away with importing from the files, not the directory/module. But that's nothing to be concerned about here - so just thanks a lot for trying.